packages/trader/src/exchange-adapters/UniswapV3Trader.ts

```diff
@@ -0,0 +1,59 @@
+import winston from "winston";
+import Web3 from "web3";
+import BigNumber from "bignumber.js";
+
+const bn = require("bignumber.js"); // Big number that comes with web3 does not support square root.
+bn.config({ EXPONENTIAL_AT: 999999, DECIMAL_PLACES: 40 });
+
+const { encodePriceSqrt } = require("@uma/common");
+const ExchangeAdapterInterface = require("./ExchangeAdapterInterface");
+const { getTruffleContract } = require("@uma/core");
+
+class UniswapV3Trader implements InstanceType<typeof ExchangeAdapterInterface> {
+  readonly tradeDeadline: number;
+  readonly UniswapV2Broker: any;
+  uniswapPair: any;
```

Comment:
nit: is this variable being used?

Reply:
it's not! looks like our linter is not working at detecting unused variables in ts.